Libraries/Opc.Ua.Server/NodeManager/MasterNodeManager.cs

```diff
@@ -459,6 +459,90 @@ public void RegisterNamespaceManager(string namespaceUri, INodeManager nodeManag
             }
         }
 
+        /// <summary>
+        /// Unregisters the node manager as the node manager for Nodes in the specified namespace.
+        /// </summary>
+        /// <param name="namespaceUri">The URI of the namespace.</param>
+        /// <param name="nodeManager">The NodeManager which no longer owns nodes in the namespace.</param>
+        /// <exception cref="ArgumentNullException">Throw if the namespaceUri or the nodeManager are null.</exception>
+        public void UnregisterNamespaceManager(string namespaceUri, INodeManager nodeManager)
+        {
+            if (String.IsNullOrEmpty(namespaceUri)) throw new ArgumentNullException(nameof(namespaceUri));
+            if (nodeManager == null) throw new ArgumentNullException(nameof(nodeManager));
+
+            // look up the namespace uri. If it is not found, we have nothing to do
+            int index = m_server.NamespaceUris.GetIndex(namespaceUri);
+            if (index < 0)
+            {
+                return;
```

Comment:
return some error here or success? at least true / false?

Reply:
Good idea as I think there are two negative paths here:

1. Namespace is unknown
2. Node Manager is not in collection

I made this change and also simplified the function body logic to avoid some unnecessary I/O.

Changes should be good for review in 0ac332d050f5bc818fdcda05a95e97775acdaae8